treeherder/perf/models.py

```diff
@@ -0,0 +1,67 @@
+from django.core.validators import RegexValidator
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from jsonfield import JSONField
+
+from treeherder.model.models import MachinePlatform, OptionCollection, Repository
+
+
+@python_2_unicode_compatible
+class PerformanceFramework(models.Model):
+    name = models.SlugField(max_length=255L, unique=True)
+    description = models.TextField()
+    url = models.URLField()
```

Comment:
> I'll just take description and url out, as we're not using them at the moment. I think we've learned that adding stuff just because we might need it later isn't a great idea. :)

+1 :-)